specification/deviceprovisioningservices/resource-manager/Microsoft.Devices/preview/2021-02-01-preview/iotdps.json

```diff
@@ -0,0 +1,2622 @@
+        ],
+        "summary": "Update an existing provisioning service's tags.",
+        "description": "Update an existing provisioning service's tags. to update other fields use the CreateOrUpdate method",
+        "x-ms-long-running-operation": true,
+        "operationId": "IotDpsResource_Update",
+        "x-ms-examples": {
+          "DPSPatch": {
+            "$ref": "./examples/DPSPatch.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionId"
+          },
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "Resource group identifier."
+          },
+          {
+            "name": "provisioningServiceName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "Name of provisioning service to create or update."
+          },
+          {
+            "name": "ProvisioningServiceTags",
+            "in": "body",
+            "required": true,
+            "description": "Updated tag information to set into the provisioning service instance.",
+            "schema": {
+              "$ref": "#/definitions/TagsResource"
+            }
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Provisioning Service was successfully updated",
+            "schema": {
+              "$ref": "#/definitions/ProvisioningServiceDescription"
+            }
+          }
+        },
+        "produces": [
+          "application/json"
+        ],
+        "consumes": [
+          "application/json"
+        ]
+      },
+      "delete": {
+        "tags": [
+          "DELETE"
+        ],
+        "summary": "Delete the Provisioning Service",
+        "description": "Deletes the Provisioning Service.",
+        "x-ms-long-running-operation": true,
+        "operationId": "IotDpsResource_Delete",
+        "x-ms-examples": {
+          "DPSDelete": {
+            "$ref": "./examples/DPSDelete.json"
+          }
+        },
+        "consumes": [],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "provisioningServiceName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "Name of provisioning service to delete."
+          },
+          {
+            "$ref": "#/parameters/subscriptionId"
+          },
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "Resource group identifier."
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          }
+        ],
+        "responses": {
+          "202": {
```

Comment:
Please consider adding "headers" property to specify Location and Azure-AsyncOperation header in the response and also update the example to show the headers there as well.

Reply:
Temporarily resolved. See details in this [comment](https://github.com/Azure/azure-rest-api-specs/pull/12378#discussion_r560402312)